docs/api/web-contents.md

```diff
@@ -590,6 +590,25 @@ Emitted when `webContents` wants to do basic auth.
 
 The usage is the same with [the `login` event of `app`](app.md#event-login).
 
+#### Event: 'dialog'
+
+Returns:
+
+* `event` Event
+* `details` Object
```

Comment:
This appears still to be missing `frameProcessId` and `frameRoutingId` added to address https://github.com/electron/electron/pull/34466#pullrequestreview-999353982 - that should be added here as well as a note that those can be used to construct a `webFrame` by calling `webFrameMain.fromId()`